### ISSUE TITLE
Metadata: Update not triggering with modifying comment bodies

### ISSUE THREAD

**Issue by GuiltyDolphin** [duckduckgo/community-platform#1382]
It looks like modifying the body of issues/comments isn't triggering the metadata to update (it is [supported](https://developer.github.com/v3/activity/events/types/#issuecommentevent)).

For example, https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2105 previously did not link to the Calculator's IA page, and did not show up on the IA page and only appeared in the 'other' issues on the Contributor Profile (it did not appear under the IA-specific issues).

After updating the issue to reference the IA page, waiting, and doing hard-refereshes, the status has not updated on the IA page or profile. This is also the case with many other IAs.

/cc @alohaas @jdorweiler

**Comment by alohaas** [comment]
@MariagraziaAlastra @jagtalon do either of you have input on this issue? 
Let us know if you need more description or screen shots.

**Comment by MariagraziaAlastra** [comment]
@GuiltyDolphin that's because we look for issues and PRs opened/closed/updated (where updated means new comments) within the last hour, and just changing the description would still make the issue fall off the radar.
I closed it and reopened it, this should be enough to update the IA Page whenever ghIssues.pl will run next time.
And since that's how we get issues linked to IAs on the profiles as well, it should reflect there too when the data gets generated again by the cronjob :smile:

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra Check out https://duckduckhack.com/u/guiltydolphin/ (go to the issues/PRs section, at the bottom); there's a *bunch* of PRs that need updating (I've added the links).

Perhaps it might be worth running something over old issues that were created before we started adding the IA page as given (after updating them ofc)?

**Comment by MariagraziaAlastra** [comment]
Right, there isn't a specific period of time we can tackle, or not limited enough that we can hope not to bust our GH API limit, that's the problem.

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra Does changing labels update it? If so I can just add/remove labels when the page has been updated.

**Comment by MariagraziaAlastra** [comment]
Nope unfortunately
